<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="59" started-at="2018-11-19T23:39:23Z" finished-at="2018-11-19T23:39:23Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="59" started-at="2018-11-19T23:39:23Z" finished-at="2018-11-19T23:39:23Z">
      <class name="JavaTesting.EvenorOddTest">
        <test-method status="PASS" signature="testEven()[pri:0, instance:JavaTesting.EvenorOddTest@7c75222b]" name="testEven" duration-ms="15" started-at="2018-11-19T18:39:23Z" finished-at="2018-11-19T18:39:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEven -->
        <test-method status="PASS" signature="testOdd()[pri:0, instance:JavaTesting.EvenorOddTest@7c75222b]" name="testOdd" duration-ms="1" started-at="2018-11-19T18:39:23Z" finished-at="2018-11-19T18:39:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOdd -->
        <test-method status="FAIL" signature="testzero()[pri:0, instance:JavaTesting.EvenorOddTest@7c75222b]" name="testzero" duration-ms="2" started-at="2018-11-19T18:39:23Z" finished-at="2018-11-19T18:39:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[0 Is Even expected [0 Is neither even nor odd] but found [Checking even number]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 0 Is Even expected [0 Is neither even nor odd] but found [Checking even number]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at JavaTesting.EvenorOddTest.testzero(EvenorOddTest.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testzero -->
      </class> <!-- JavaTesting.EvenorOddTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
